components/cascader/nz-cascader.component.html

```diff
@@ -78,7 +78,7 @@
         (mouseleave)="onOptionMouseLeave(option, i, $event)"
         (click)="onOptionClick(option, i, $event)">
       </li>
-      <li *ngIf="isSearching"
+      <li *ngIf="isSearching && !columns[0].length"
```

Comment:
columns[0] may be null?

Reply:
@vthinkxie This is secured here

https://github.com/NG-ZORRO/ng-zorro-antd/blob/07f76af35690417571ca6c32a947f3b7b260ffc6/components/cascader/nz-cascader.component.ts#L715

And I wrote a test about it.

Reply:
https://github.com/NG-ZORRO/ng-zorro-antd/blob/07f76af35690417571ca6c32a947f3b7b260ffc6/components/cascader/nz-cascader.component.ts#L130
should be
`columns: CascaderOption[][] = [ [] ];`
?

Reply:
If so there would be an blank dropdown.

Reply:
fine...